modules/preview-react/information-highlight/lib/Button.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
```

Comment:
So for this, a link is the right file name for this. We should not be having a traditional button within this. It uses a `TertiaryButton` that looks like a link in the design.

Reply:
updated